.devops/nodo-db-migration-pipelines.yml

```diff
@@ -0,0 +1,234 @@
+pr: none
+trigger: none
+#trigger:
+#  - develop
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+parameters:
+  - name: k8sEnv
+    displayName: K8s Environment
+    type: string
+    default: dev
+    values:
+      - dev
+      - uat
+      - prd
+  - name: dbVersion
+    displayName: Database Version
```

Comment:
add comment to describe it ( ex. refers to versions current repo under `liquibase/changelog/` folder

Reply:
done